netjson-monitoring.lua

```diff
@@ -130,7 +152,8 @@ netjson = {
     resources = {
         load = load_average,
         memory = system_info.memory,
-        swap = system_info.swap
+        swap = system_info.swap,
+        disk_usage = parse_disk_usage()
```

Comment:
As @nemesisdesign mentioned, it would be very useful to also send the number of cores available in the system so that we can calculate the % of cpu use later.

Do you mind adding it in this PR too?

You can easily get it from the output of `cat /proc/cpuinfo | grep -c processor`